test/frontend/frontend.bundle.js

```diff
@@ -0,0 +1,22 @@
+class FakeFile extends Blob {
+    constructor(name, data, opt) {
+        super(data, opt);
+        this.name = name;
+    }
```

Comment:
Indenting looks weird in this file (4 space indent). We may want to confirm that ESLint and prettier are linting these files. I would have almost expected CircleCI linting to fail on these.